central/image/datastore/store/postgres/store_test.go

```diff
@@ -76,7 +76,7 @@ func (s *ImagesStoreSuite) TestStore() {
 			vuln.FirstImageOccurrence = foundImage.GetLastUpdated()
 			vuln.Epss = nil
 			// TODO(ROX-27402) remove this
```

Comment:
@dashrews78 I see this ticket is closed. Should this be deleted now?

Reply:
yes just the comment though.  Not sure how that wasn't flagged by style long time ago.